__tests__/src/screens/NationDetailsContainer/NationDetailsScreen/index.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import configureStore from 'redux-mock-store';
+
+import NationDetailsScreen from '../../../../../src/screens/NationDetailsContainer/NationDetailsScreen';
+
+describe('DetailNationScreen', () => {
+  let detailsScreen;
+
+  const initialStateMock = {
+    nations: {
+      nations: [{
+        _id: 1,
```

Comment:
Sorry, missed this one, please use `id` instead.

Reply:
No, I forget to changed it too, sorry